app/src/main/java/com/framgia/soundclound/data/model/Collection.java

```diff
@@ -0,0 +1,36 @@
+package com.framgia.soundclound.data.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports